src/core/biblio.js

```diff
@@ -1,14 +1,17 @@
 // Module core/biblio
-// Handles bibliographic references
+// Pre-processes bibliographic references
 // Configuration:
 //  - localBiblio: override or supplement the official biblio with your own.
 
 /*jshint jquery: true*/
 /*globals console*/
 import { biblioDB } from "core/biblio-db";
-import { createResourceHint, addId } from "core/utils";
+import { createResourceHint } from "core/utils";
 import { pub } from "core/pubsubhub";
 
+// for backward compatibity
+export { wireReference, stringifyReference } from "core/render-biblio";
```

Comment:
Do we know where are they used externally? Do we really need to export them? Exporting won't hurt though.